### DISCUSSION TITLE
我在用xmake.lua生成c++的dll动态库，如何支持生成的dll有版本号

### DISCUSSION THREAD

**Discussion by yisqiu** [xmake-io/xmake#2416] [General]
现在生成的dll右键属性菜单里面文件版本、文件名称和产品版本都是空的，如何把版本信息加进来，谢谢！

**Comment by waruqi** [comment]
这种自己google 下，自己加上对应 flags

**Reply by yisqiu** [reply]
> flags

能否详细一点，找了很多地方都无效，多谢

**Reply by waruqi** [reply]
加编译选项 add_cxflags 加dll链接选项 add_shflags ，其他的版本信息啥的功能特性，不归xmake管，自己研究
